tokenomics/sagittarius-engine.md

```diff
@@ -1,32 +1,52 @@
 
-Maker Governance can later add farmable SubDAO governance tokens, making the first version of the Sagittarius Engine compatible with the launch of SubDAOs.
+## Entry and Exit
+When users deposit MKR into the Sagittarius Engine, they must delegate the deposited MKR through the [Easy Governance Frontend (EGF)](../maker-core/easy-governance-frontend.md). Depositors must specify a target delegate smart contract when making their deposit.
 
-Token farming ends when the user withdraws their MKR from the Sagittarius Engine.
+When a user withdraws their MKR from the Sagittarius Engine, 15% of the deposited MKR is burned.
 
 ## MKR Vaults
-The Sagittarius Engine allows users to use deposited MKR as collateral to generate Dai, similar to normal Maker Vaults. 
+The Sagittarius Engine allows users to use deposited MKR as collateral to generate Dai, similar to normal Maker Vaults. The parameters for these vaults are below:
+
+#### [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) 
+There is no hard Debt Ceiling for these MKR vaults. However, a [Debt Ceiling Instant Access Module](https://manual.makerdao.com/module-index/module-dciam) will be configured to ensure that the Debt Ceiling does not increase too fast. A Soft Debt Ceiling parameter is set based on the value of all protocol-owned and SubDAO-owned assets.
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) 
+If the total Sagittarius Engine debt is less than the Soft Debt Ceiling, this is set to be equal to the [Dai Savings Rate](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate). It increases exponentially with the amount of Sagittarius Engine debt that exceeds the Soft Debt Ceiling.
+
+#### Price Oracle
+The oracle for MKR price uses [Elixir](elixir.md) as the primary data source. 
 
-The oracle for MKR prices used to manage these MKR vaults has constrained upwards movement. That is, Maker Governance can choose to limit the speed at which the reported price of MKR increases, regardless of how fast the price is actually increasing. This constraint does not apply to downward price movement. 
+A "sticky price" feature limits upward movement of the price within a fixed time period. That is, Maker Governance can choose to limit the speed at which the reported price of MKR increases, regardless of how fast the Elixir price is increasing. This constraint does not apply to downward price movement. 
 
-When an MKR vault is liquidated, the MKR is undelegated and put up for auction. Token farming also stops when a vault is liquidated.
+#### [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio)
+The Liquidation Ratio is set to 200\%. If a vault falls below this collateralization ratio, it is liquidated completely using Elixir order books.
```

Comment:
Backslash for percentage?

Reply:
Latex hangover. Apparently markdown doesn't need the backslash!